main.css

```diff
@@ -212,3 +212,19 @@ section {
     padding: 1em;
     text-decoration: none;
 }
+
+
+::-webkit-scrollbar {
+    width: 10px;
```

Comment:
Tohle nechat `auto`, ať si prohlížeč poradí se šířkou sám na různých monitorech. Fixní pixely budou vypadat různě blbě na malý obrazovce vs na 4K UHD.

Zároveň tenhle styling nastav i pro Firefox. Sice to tam nepůjde nastavit jedna ku jedné, ale nějaké `scrollbar-color: var(--yellow) var(--blue);` by mohlo pomoct. Ať jsme aspoň trochu konzistentní napříč prohlížeči.

Reply:
Tak ten Firefox nechám na tobě, on Firefox mění barvy při kliknutí a vypadá to celkem hnusně.

Reply:
It looks too wide when it is auto, I checked it on my 4k screen, looks kinda fine tho

Reply:
in fact, the screenshot I added is on 4k screen